### DISCUSSION TITLE
Photos ordered by moment taken and not by filename?

### DISCUSSION THREAD

**Discussion by kikendo** [immich-app/immich#13550] [Q&A]
I noticed that the photos on my timeline are ordered by filename within each day, for example, if I have photos on one day with prefix IMG**** they will be shown after photos prefixed DSC****

This is really annoying and would like for photos to be ordered just by date taken. is this possible at all or is this a feature request?
I actually don't know why this would be the case as I never had a photo library that did this, photos have always been shown to me ordered by moment taken in any library I have used before.

**Comment by mmomjian** [comment]
This is not a thing. It's probably something to do with the way the time/timezone is set on each device.

**Reply by kikendo** [reply]
OK I just paid a little bit more attention and this does not happen on my timeline, but on the "X year(s) ago" section. 
That section has the photos in the wrong order time-wise. It's not the filename, but the order is wrong.

On the timeline, all the photos are aligned correctly, even when they were taken at different timezones.

**Comment by kikendo** [comment]
Submitted as an issue https://github.com/immich-app/immich/issues/13566
Because this is just odd.
